guides/request-validation-express/example-1/example-1.3.x.js

```diff
@@ -0,0 +1,35 @@
+// You can find your Twilio Auth Token here: https://www.twilio.com/console
+// Which can be set at runtime as follows:
+// $ TWILIO_AUTH_TOKEN=XXXXXXXXXXXXXXXXXXX node index.js
+// Please note that this will not work unless you set the TWILIO_AUTH_TOKEN
+// environment variable.
+var twilio = require('twilio');
+var app = require('express')();
+var bodyParser = require('body-parser');
+var TwimlResponse = twilio.TwimlResponse;
+
+app.use(bodyParser.urlencoded({extended: false}));
+
+app.post('/voice', twilio.webhook(), function(req, res) {
```

Comment:
What do you think about using arrow functions to keep consistency with the use of the ES6 syntax? We're already using template literals, right?